### PR TITLE
Update MIME Types for font-face 

### PR DESCRIPTION
**Pull request by c01nd01r**
Hi!
According with the specifications of W3C, WOFF / WOFF2 fonts have certain MIME types
WOFF: application/font-woff (https://www.w3.org/TR/WOFF/#appendix-b)
WOFF2: font/woff2 (https://www.w3.org/TR/WOFF2/#IMT)

For correct generation of naming MIME types in font-face rule, I made small changes.
Also updated tests and demo. 
What do you think? :)

### CODE REVIEW

File: lib/fontoptim.js [hunk 1]
```diff
@@ -62,12 +62,22 @@ FontOptim.prototype = {
...
+		var mime = this.getMimeType(format);
 		return _.template(this.fontUriTemplate, {
 			format: format,
+			mime: mime,
 			base64: base64
 		});
 	},
 
+	getMimeType: function(format) {
+		var mimeFonts = {
```
Comment (sapegin) [inline lib/fontoptim.js]: I’d move this const to the top of the file.

**Review by sapegin — COMMENTED** [review]
Looks good, thanks!

File: lib/fontoptim.js (hunk 1 above)
Comment (c01nd01r) [reply lib/fontoptim.js]: Good remark, will be done.

**Comment by c01nd01r** [comment]
Thank you for a good tool :)

**Pull request merged**